syncer/db_test.go

```diff
@@ -32,3 +33,18 @@ func (s *testSyncerSuite) TestGetServerID(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(id, Greater, int64(0))
 }
+
+func (s *testSyncerSuite) TestBinaryLogs(c *C) {
+	files, err := getBinaryLogs(s.db)
+	c.Assert(err, IsNil)
+	c.Assert(files, Not(HasLen), 0)
+
+	pos := mysql.Position{
+		Name: files[len(files)-1].name,
+		Pos:  0,
+	}
+
+	remainingSize, err := countBinaryLogsSize(pos, s.db)
+	c.Assert(err, IsNil)
+	c.Assert(remainingSize, Equals, files[len(files)-1].size)
```

Comment:
execute [FLUSH BINARY LOGS](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-binary-logs) and to verify `getBinaryLogs`/`countBinaryLogsSize` again?

Reply:
done